rpg_game.py

```diff
@@ -7,6 +7,64 @@
 """
 import random
 import time
+import re
+
+
+
+class InputManager(object):
+    """Class with methods for sanitizing user input."""
+    def __init__(self):
+        pass
+
+
+    def get_yes_or_no(prompt):
+        """Sanatizes user input for yes or no questions.
+
+        Args:
+            prompt: the question displayed to the user.
+
+        Returns:
+            Bool - True for yes answer, False for No Answer."""
+
+        while True:
+            print(prompt)
+            ans = input("> ")
+            if ans[0].lower() == "y":
+                return True
+            elif ans[0].lower() == "n":
+                return False
+            else:
+                print("That's not a valid answer, bud.")
+
+
+    def get_numerical_input(prompt, num_options, extra_option=None):
+        """Sanatizes user input when chosing between multiple options.
+
+        Args:
+            prompt: the question displayed to the user
+            num_options: the amount of options the user has to chode between.
+            extra_options: optional arguement if there is an extra argument
+            outside of 'num_options'
+
+        Returns:
+            ans: sanitised answer to the prompt."""
+        while True:
+            print(prompt)
+            try:
+                ans = int(input("> "))
+            except ValueError:
+                print("You need to enter a number, fella.")
+                continue
+            if not extra_option:
+                if re.match(f"^[1-{str(num_options)}]$", str(ans)):
+                    return ans
+                else:
+                    print("That's not a valid answer, bud.")
+            else:
+                if ans == extra_option:
+                    return ans
+                else:
+                    print("That's not a valid answer, bud.")
```

Comment:
I created the input manager to handle the different types of input you ask of the user. Yes and No questions and questions where the user inputs a number. 

The only tricky thing was with the numbers I had the `get_numerical_input()` method take an optional argument for the case of `10` to allow the user to leave the store.